EIPS/eip-1.md

```diff
@@ -67,54 +87,112 @@ These calls generally result in a "rough consensus" around what EIPs should be i
 
 *In short, your role as the champion is to write the EIP using the style and format described below, shepherd the discussions in the appropriate forums, and build community consensus around the idea.*
 
+
 ### EIP Process
 
 The following is the standardization process for all EIPs in all tracks:
 
 ![EIP Status Diagram](../assets/eip-1/EIP-process-update.jpg)
 
-**Idea** - An idea that is pre-draft. This is not tracked within the EIP Repository.
+#### Idea
 
-**Draft** - The first formally tracked stage of an EIP in development. An EIP is merged by an EIP Editor into the EIP repository when properly formatted.
+An idea that is pre-draft. This is not tracked within the EIP Repository.
 
-**Review** - An EIP Author marks an EIP as ready for and requesting Peer Review.
+#### Draft
 
-**Last Call** - This is the final review window for an EIP before moving to `Final`. An EIP editor will assign `Last Call` status and set a review end date (`last-call-deadline`), typically 14 days later.
+The first formally tracked stage of an EIP in development. An EIP is merged by an EIP Editor into the EIP repository when it meets some initial requirements: It should be technically sound, the Title and Preamble must be correct, and all included Sections should be readable.  Problems with language like spelling, grammar, markup, etc. are only considered warnings.
 
-If this period results in necessary normative changes it will revert the EIP to `Review`.
+#### Review
 
-**Final** - This EIP represents the final standard. A Final EIP exists in a state of finality and should only be updated to correct errata and add non-normative clarifications.
+An EIP Author marks an EIP as ready for and requesting Peer Review.  An Editor will merge the EIP when it meets all EIP requirements: it should be a technically sound, complete specification, properly formatted, with correct language, markup and external references.
+
+#### Last Call
+
+This is the final review window for an EIP before moving to `Final`. An EIP editor will assign `Last Call` status and set a review end date (`last-call-deadline`), typically 14 days later.
+
+*If this period results in necessary normative changes it will revert the EIP to `Review`.*
```

Comment:
```suggestion
This is the final review window for an EIP before moving to `Final`. An EIP Author may request `Last Call` status and set a review end date (`last-call-deadline`), typically 14 days later.

*If this period results in necessary normative changes it will revert the EIP to `Review`.*
```

Reply:
I'm not sure why you want to change this.  The Author may request Last Call, but it seems like an Editors job to make the move.  But I'm not  that concerned either way.

Reply:
I've always operated under the assumption that authors request a status change, and editors grant it.

I'm concerned that the wording "[a]n EIP editor will assign `Last Call` status" makes it sound like authors don't need to open a pull request. Without a pull request, I don't see how a proposal can move to last call.

Reply:
if part of the problem is public perception of editorial omnipotence, it never hurts to *phrase* things less editor-centrically, as an author-push not an editor-pull :D